src/app/about/page.js

```diff
@@ -95,21 +107,33 @@ const AboutPage = () => {
             options={OPTIONS}
           />
 
-          <br />
-          <br />
-          <h1 id="impact">More Info</h1>
-          <br />
-          <div className="flex flex-nowrap items-center justify-center ">
+          {/* <br />
+                    <br />
+                    <h1 id="impact">More Info</h1>
+                    <br />
+                    <div className="flex flex-nowrap items-center justify-center ">
+                        <Link className="bg-NavBarColor rounded-full text-white mr-4 py-3 px-6 transition ease-in-out duration-200 hover:bg-black" href="/about/projects">
+                            <Button >All Projects</Button>
+                        </Link>
+                        <br />
+                        <Link className="bg-UGA rounded-full text-white ml-4 py-3 px-6 transition ease-in-out duration-200 hover:bg-black" href="/about/people">
+                            <Button >Our People</Button>
+                        </Link>
+                    </div> */}
```

Comment:
```suggestion
```
Please don't do this. We have source control to keep track of previous versions.